src/app/components/MediaLoader/README.md

```diff
@@ -0,0 +1,22 @@
+# Media Player
+
+## Description
+
+This component loads a media player into our webpage. It uses the BBC Universal Media Player module (BUMP) to choose the correct the Embedded Media Player (EMP) for the user's device. Using BUMP removes considerations towards variations in user device and web-browsers as BUMP handles this logic for us and provides the best fitting EMP for our webpage. For example, BUMP can cater for differences between mobiles, laptops and desktops, but also factors such as whether the user is using network data (3G) or a WIFI. 
+
+
+## Local Development
+So that the EMP can load video data, our localhost's domain name should be altered from `localhost:7080` to `localhost.bbc.com:7080` to fully encorporate the bbc domain name (bbc) and top-level domain (.com).
+
+To do, run the command: 
```

Comment:
Something along these lines, so that folk don't run it multiple times?
```suggestion
First, run the command (you only need to ever run this once): 
```